tox.ini

```diff
@@ -40,22 +40,22 @@ sources = setup.py src/ docs/conch/examples docs/mail/examples docs/names/exampl
 # process to generate its own wheels.
 wheel = True
 parallel_show_output = True
-;; dependencies managed by extras in setup.cfg
+;; dependencies managed by extras in pyproject.toml
 extras =
     ; The "nodeps" build still depends on PyHamcrest.
     nodeps: test
 
-    ; We also need to include `dev_release` so that we can test our
-    ; release helpers or documentation generation, and `conch_nacl` so that
+    ; We also need to include `dev-release` so that we can test our
+    ; release helpers or documentation generation, and `conch-nacl` so that
     ; we can test conch fallbacks to PyNaCl without requiring most users to
     ; install it.
-    alldeps: all_non_platform, dev_release, conch_nacl
+    alldeps: all-non-platform, dev-release, conch-nacl
```

Comment:
can we continue to use the old extra names here,

just as a test that they continue to work?